### ISSUE TITLE
Attention

### ISSUE THREAD

**Issue by Massacre96Wj** [mgh5212819/Chinese-Marketing-text-generation#5]
decoder_emb = self.embedding(x_t)
decoder_output, decoder_states = self.lstm(decoder_emb, decoder_states)
#concatenate context vector and decoder state
#(batch_size, 3*hidden_units)
decoder_output = decoder_output.view(-1, config.hidden_size)
concat_vector = torch.cat(
    [decoder_output,
     context_vector],
    dim=-1)
请问你的attention机制采用的是把得到的context_vectors和lstm的输出decoder_output进行结合，再经过mlp，
我理解attention应该是将context_vectors和输入结合再经过lstm得到输出decoder_output。
